schema/pvcollada_schema_0.1.xsd

```diff
@@ -755,45 +755,51 @@ xmlns:collada="http://www.collada.org/2008/03/COLLADASchema">
       </xs:attribute>
     </xs:complexType>
   </xs:element>
-  <xs:element name="modules">
+  <xs:element name="components">
```

Comment:
this `components` wrapper is present in the example file, but not in the schema.

So I added it here.
Feedback welcome.

Reply:
Thanks @Michalzr, <components> came up after this section was drafted in the xsd. Good catch.

I think we want a different design for <components>:

```
  <xs:element name="components">
    <xs:complexType>
      <xs:all>
        <xs:element name="modules" minOccurs="0" />
        <xs:element name="inverters" minOccurs="0" />
...
      </xs:all>
    </xs:complexType>
```

and in the definition of each of the plural blocks `modules`, `inverters`, etc., change `minOccurs="0"` to `minOccurs="1"` where `minOccurs="0"` is present.

This way, `components` contains 0 or 1 of each of the plural blocks, in any order. And a plural block must have at least one entry.

Reply:
Good idea. Updated.

I also removed every instance of `minOccurs="1"` from the schema, as `1` is the default.